src/test/java/api/requests/AllowedServicePointsAPITests.java

```diff
@@ -81,96 +89,268 @@ void getFailsWithMultipleErrors() {
       "Instance ID is not a valid UUID: instanceId. Item ID is not a valid UUID: itemId."));
   }
 
+  public static Object[] shouldReturnListOfAllowedServicePointsForRequestParameters() {
+    String sp1Id = randomId();
+    String sp2Id = randomId();
+    var sp1 = new AllowedServicePoint(sp1Id, "SP One");
+    var sp2 = new AllowedServicePoint(sp2Id, "SP Two");
+
+    List<AllowedServicePoint> none = List.of();
+    List<AllowedServicePoint> oneAndTwo = List.of(sp1, sp2);
+
+    return new Object[][]{
+      {RequestType.PAGE, RequestLevel.ITEM, ItemStatus.AVAILABLE, oneAndTwo, oneAndTwo},
+      {RequestType.HOLD, RequestLevel.ITEM, ItemStatus.AVAILABLE, oneAndTwo, none},
+      {RequestType.RECALL, RequestLevel.ITEM, ItemStatus.AVAILABLE, oneAndTwo, none},
+      {RequestType.PAGE, RequestLevel.TITLE, ItemStatus.AVAILABLE, oneAndTwo, oneAndTwo},
+      {RequestType.HOLD, RequestLevel.TITLE, ItemStatus.AVAILABLE, oneAndTwo, none},
+      {RequestType.RECALL, RequestLevel.TITLE, ItemStatus.AVAILABLE, oneAndTwo, none},
+      {RequestType.PAGE, RequestLevel.ITEM, ItemStatus.CHECKED_OUT, oneAndTwo, none},
+      {RequestType.HOLD, RequestLevel.ITEM, ItemStatus.CHECKED_OUT, oneAndTwo, oneAndTwo},
+      {RequestType.RECALL, RequestLevel.ITEM, ItemStatus.CHECKED_OUT, oneAndTwo, oneAndTwo},
+      {RequestType.PAGE, RequestLevel.TITLE, ItemStatus.CHECKED_OUT, oneAndTwo, none},
+      {RequestType.HOLD, RequestLevel.TITLE, ItemStatus.CHECKED_OUT, oneAndTwo, oneAndTwo},
+      {RequestType.RECALL, RequestLevel.TITLE, ItemStatus.CHECKED_OUT, oneAndTwo, oneAndTwo},
+    };
+  }
+
   @ParameterizedTest
-  @MethodSource("parameters")
+  @MethodSource("shouldReturnListOfAllowedServicePointsForRequestParameters")
   void shouldReturnListOfAllowedServicePointsForRequest(RequestType requestType,
-    RequestLevel requestLevel) {
+    RequestLevel requestLevel, ItemStatus itemStatus, List<AllowedServicePoint> allowedSpByPolicy,
+    List<AllowedServicePoint> allowedSpInResponse) {
 
     var requesterId = usersFixture.steve().getId().toString();
-    var itemId = itemsFixture.basedUponNod().getId().toString();
-    var instanceId = itemsFixture.createMultipleItemsForTheSameInstance(2).get(0)
-      .getInstanceId().toString();
-    var cd1 = servicePointsFixture.cd1();
-    var cd2 = servicePointsFixture.cd2();
-    setRequestPolicyWithAllowedServicePoints(requestType, cd1.getId(), cd2.getId());
+    var items = itemsFixture.createMultipleItemForTheSameInstance(1,
+      List.of(ib -> ib.withStatus(itemStatus.getValue())));
```

Comment:
How about something simpler?
```suggestion
    var item = itemsFixture.basedUponNod(ib -> ib.withStatus(itemStatus.getValue()))
```